### PR TITLE
Update profile page

### PR DESCRIPTION
**Pull request by kayleenovak**
## Description:

This PR addresses issue #46  #45 .

The changes made to the codebase in this PR:

   - User can edit profile - edit functionality is conditional
   - User can see contact information for only their connections

## Testing

These changes did not affect the existing test suite.

## Requests for review

Areas of concern:
* There are a lot of conditionals in the profile page. Is there anything you would suggest to make it a bit cleaner?

### CODE REVIEW

File: ProfilePage.js [hunk 1]
```diff
@@ -15,34 +15,73 @@ export class ProfilePage extends Component {
...
+                <Icon name='linkedin' size={ 20 } color='#4AA9C5' style={ {marginRight: 5} }/>
+                <Text>{ user.linkedin }</Text>
+              </View>
+              <View style={ styles.iconContainer }>
+                <Icon name='github' size={ 20 } color='#4AA9C5' style={ {marginRight: 5} }/>
+                <Text>{ user.github }</Text>
+              </View>
               <Text style={ styles.bio }>{ user.bio }</Text>
               {
                 this.props.user !== user && <TouchableHighlight style={styles.connectBtn}><Button title='Connect' color='white' /></TouchableHighlight>
```
Comment (KylieStanley) [inline ProfilePage.js]: Could we set this as a variable and put it at the top with the other ternaries to add to the consistency?

File: ProfilePage.js [hunk 2]
```diff
@@ -15,34 +15,73 @@ export class ProfilePage extends Component {
...
-            <Text style={ styles.connections }>Connections</Text>
+            {
+              connectionInfo
+            }
+            <Text style={ styles.connections}>Connections</Text>
             <View style={ styles.connectionsContainer }>
               {
                 user.connections && user.connections.map((connection) => {
-                  return <Connection id={ connection.id } viewProfile={ this.viewProfile } />
+                  return <Connection connection={ connection } viewProfile={ this.viewProfile } />
```
Comment (KylieStanley) [inline ProfilePage.js]: I think this one could be added to the variables at the top of render as well!

**Review by KylieStanley — CHANGES_REQUESTED** [review]
Everything looks good! I just left a couple comments for minor changes.

File: ProfilePage.js (hunk 1 above)
Comment (kayleenovak) [reply ProfilePage.js]: I took the functionality out of the return 😄

**Review by KylieStanley — APPROVED** [review]

**Comment by KylieStanley** [comment]
Awesome thanks for making those changes, merging in.

**Pull request merged**